Gulpfile.js

```diff
@@ -107,6 +112,7 @@ const projects = [
         cwd: "packages/table/",
         dependencies: ["core"],
         copy: false,
+        isotest: true,
```

Comment:
yeah that's why i suggested changing to `tests` so we'd have `test-karma-core` and `test-iso-core`.